subsys/shell/Kconfig.backends

```diff
@@ -0,0 +1,25 @@
+# Kconfig.backends - Shell badckends configuration options
+
+#
+# Copyright (c) 2018 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig SHELL_BACKENDS
+	bool "Enable shell backends"
+	default y
+	help
+	  "Enable shell backends."
+
+if SHELL_BACKENDS
```

Comment:
shouldn't this be a choice to enable adding more backends in the future?

Reply:
@nashif : Wouldn't `choice` limit us to only 1 backend at a time?

Reply:
> @nashif : Wouldn't `choice` limit us to only 1 backend at a time?

correct, I misread the PR comment above, so that should be fine as is.